### ISSUE TITLE
When choosing 'custom' as dataset, Accuracy keeps constant,no matter what I do?!

### ISSUE THREAD

**Issue by rosefun** [blackecho/Deep-Learning-TensorFlow#67]
By installing the yadlt,I choose the` 'custom' `,and load my data(TE data) as the dataset .

The result is as follow:

```
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
Accuracy: 0.438:
```

`Test set accuracy: 0.3919999897480011`

What I did:

 I ran the project many times, the `accuracy `could't  changed,staying the same number,which confused me.

However, when I choosed the 'MNIST' as the dataset, the accuracy could work,and changed every time.

What's more, when I changed `the rbm_layers`, the `construction loss` could change, but `the set accuracy `still stayed 0.391999897....

Amazing,  I even think `the set accuracy` maybe a constant!

Something is wrong, but I can't find it!

Hopefully for your reply !

Best wishes !

**Comment by rosefun** [comment]
Maybe it's caused by Nan when Hyperparameter is not setted correctly.
